RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -752,6 +761,20 @@ bool MuonIdProducer::isGoodRPCMuon( const reco::Muon& muon )
   return ( muon.numberOfMatchedRPCLayers( reco::Muon::RPCHitAndTrackArbitration ) > minNumberOfMatches_ );
 }
 
+bool MuonIdProducer::isGEMMuon( const reco::Muon& muon )
+{
+  // need to update min cuts on pt
```

Comment:
GEM is in the same acceptance as CSC.
So, the same p/pt cuts should still  work.